src/commands/server/start.ts

```diff
@@ -54,94 +62,150 @@ export default class Start extends Command {
-      { title: 'Verify if Tiller service exist', task: async (ctx: any) => { ctx.tillerServiceExist = await helm.tillerServiceExist() } },
-      { title: 'Create Tiller Service', skip: (ctx: any) => { if (ctx.tillerServiceExist) { return 'Tiller Service already exist.' } }, task: () => helm.createTillerService() },
-      // { title: 'Pre-pull Che server image', skip: () => 'Not implemented yet', task: () => {}},
-      { title: `Verify if namespace ${flags.chenamespace} exist`, task: async (ctx: any, task: any) => { if (ctx.cheNamespaceExist = await che.cheNamespaceExist(flags.chenamespace)) { task.title = `${task.title}...It does.` } else { task.title = `${task.title}...It doesn't.` } } },
-      { title: 'Verify if Che server is already running', skip: (ctx: any) => { if (!ctx.cheNamespaceExist) { ctx.isCheRunning = false; return 'Che namespace doesn\'t exist.' } }, task: async (ctx: any) => { ctx.isCheRunning = await che.isCheServerReady(flags.chenamespace) } },
-      { title: 'Deploy Che Server', skip: (ctx: any) => { if (ctx.isCheRunning) { return 'Che is already running.' } }, task: () => this.deployChe(flags) },
-      // { title: 'Waiting for Che Server pod to be created', skip: () => 'Not implemented yet', task: () => {}},
-      { title: 'Waiting for Che Server to start and respond', skip: (ctx: any) => { if (ctx.isCheRunning) { return 'Che is already running.' } }, task: () => che.isCheServerReady(flags.chenamespace, bootTimeout) },
-      { title: 'Retrieving Che Server URL', task: async (ctx: any, task: any) => { ctx.cheURL = await che.cheURL(flags.chenamespace); task.title = await `${task.title}...${ctx.cheURL}` } },
-      // { title: 'Open Che Server Dashboard in browser', enable: () => false /* Doesn\'t work when chectl is packaged with zeit/pkg */, task: async (ctx: any) => { process.platform === 'linux' ? await cli.open(ctx.cheURL, { app: 'xdg-open' }) : await cli.open(ctx.cheURL, { app: 'open' }) }}
-    ], {
-      renderer: listr_renderer
-    })
+    let ingressName = 'che-ingress'
 
-    try {
-      await tasks.run()
-    } catch (err) {
-      this.error(err)
+    // Platform Checks
+    let platformCheckTasks = new Listr()
+    if (flags.platform === 'minikube') {
+      platformCheckTasks = new Listr([{
+        title: 'Platform preflight checklist (minikube) ✈️',
+        task: () => minikube.startTasks(this)
+      }], {renderer: listr_renderer, collapse: false})
+      if (!flags.domain) {
+        const { stdout } = await execa.shell('minikube ip')
+        flags.domain = stdout + '.nip.io'
+      }
+    } else {
+      this.error(`Platform ${flags.installer} is not supported yet ¯\\_(ツ)_/¯`)
+      this.exit()
     }
 
-    notifier.notify({
-      title: 'chectl',
-      message: 'Command server:start has completed successfully.'
+    // Installer
+    let installerTasks = new Listr()
+    if (flags.installer === 'helm') {
+      installerTasks = new Listr([{
```

Comment:
no need to reinitialize `installerTasks`, it's already created
just `installerTasks.add({...})` and pass the option to the constructor

Reply:
Thanks. Updated